### PR TITLE
Feature/current-location-updated-directions-page

### PR DESCRIPTION
**Pull request by ChesterLY**
This PR closes issue #10 
- Changed the toggle buttons
- Added camera zoom to start
- Starting point defaults to current location in the direction page

### CODE REVIEW

**Comment by MrJCipherButtles** [comment]
I added moving the google places API session token management in a more appropriate place, so, that sessions are properly managed when getting suggested places for directions

**Comment by MrJCipherButtles** [comment]
@EspressoCode your wishes have been answered to 😄

File: lib/global.dart [hunk 1]
```diff
@@ -5,3 +5,7 @@ disabilityMode is to save the state of the toggle from the app drawer.
 it should be removed once a config file is used to save the user preferences
```
Comment (DiveshPatel) [inline lib/global.dart]: We will need to replace the globals with a DirectionsNotifier class

File: lib/global.dart [hunk 2]
```diff
@@ -5,3 +5,7 @@ disabilityMode is to save the state of the toggle from the app drawer.
...
 */
 bool disabilityMode = false;
```
Comment (Mehrdadmaskull) [inline lib/global.dart]: Why not put the travelMode in goto_page_new.dart? Otherwise LGTM

File: lib/global.dart (hunk 1 above)
Comment (MrJCipherButtles) [reply lib/global.dart]: You can use the global or change it to a full class that deals with TravelModeNotifier, if you deem more beneficial. That's an implementation detail for those implementing the directions 👍

File: lib/global.dart (hunk 2 above)
Comment (MrJCipherButtles) [reply lib/global.dart]: Like the disabilityMode, the travelMode is a setting/preference that is needed outside the scope of the goto_page_new.dart. A global is easily accessible, but yeah it's an implementation detail that could possibly be more restrictive, if a need to do so arises

**Pull request merged**